src/App.vue

```diff
@@ -525,11 +526,11 @@ export default {
 
       const draft = await LegalServices.fetchDraftApp(this.tempRegNumber)
 
-      // Handle Draft filings
+      // handle draft appliation
       this.storeDraftApp(draft)
 
-      // if the draft has a NR, load it
-      if (this.localNrNumber) {
+      // if this app is a task (not a filing), and it has a NR, load it
+      if (this.isAppTask && this.localNrNumber) {
```

Comment:
Here, we are loading the bootstrap business using its Txxx number (not BCxxx, FMxxx, etc).

Here, there is special handling -- since the business does not truly exist, we cannot fetch business info, addresses, parties, etc -- instead, we fetch the application filing, which is either a Draft (before being filed) or Pending or Completed. In the former case, we add the filing to the Todo List (it's an application task). In the latter case, we add the filing to the Recent Filing History List (it's an application filing).

I added this new check here since we only need to fetch the NR if this is an application task (in Todo List), since we display the NR info there. We don't need to fetch the NR once the application has been filed.